examples/digital_fingerprinting/production/morpheus/dfp_azure_modules_inference.py

```diff
@@ -0,0 +1,310 @@
+from morpheus.utils.module_ids import MODULE_NAMESPACE
+
+
+@click.command()
+@click.option(
+    "--skip_user",
+    multiple=True,
+    type=str,
+    help="User IDs to skip. Mutually exclusive with only_user",
+)
+@click.option(
+    "--only_user",
+    multiple=True,
+    type=str,
+    help="Only users specified by this option will be included. Mutually exclusive with skip_user",
+)
+@click.option(
+    "--start_time",
+    type=click.DateTime(
+        formats=['%Y-%m-%d', '%Y-%m-%dT%H:%M:%S', '%Y-%m-%d %H:%M:%S', '%Y-%m-%dT%H:%M:%S%z', '%Y-%m-%d %H:%M:%S%z']),
+    default=None,
+    help="The start of the time window, if undefined start_date will be `now()-duration`",
+)
+@click.option(
+    "--duration",
+    type=str,
+    default="1d",
+    help="The duration to run starting from start_time",
+)
+@click.option(
+    "--cache_dir",
+    type=str,
+    default="./.cache/dfp",
+    show_envvar=True,
+    help="The location to cache data such as S3 downloads and pre-processed data",
+)
+@click.option("--log_level",
+              default=logging.getLevelName(Config().log_level),
+              type=click.Choice(get_log_levels(), case_sensitive=False),
+              callback=parse_log_level,
+              help="Specify the logging level to use.")
+@click.option("--sample_rate_s",
+              type=int,
+              default=0,
+              show_envvar=True,
+              help="Minimum time step, in milliseconds, between object logs.")
+@click.option(
+    "--input_file",
+    "-f",
+    type=str,
+    multiple=True,
+    help=("List of files to process. Can specify multiple arguments for multiple files. "
+          "Also accepts glob (*) wildcards and schema prefixes such as `s3://`. "
+          "For example, to make a local cache of an s3 bucket, use `filecache::s3://mybucket/*`. "
+          "See fsspec documentation for list of possible options."),
+)
+@click.option('--tracking_uri',
+              type=str,
+              default="http://mlflow:5000",
+              help=("The MLflow tracking URI to connect to the tracking backend."))
+def run_pipeline(skip_user: typing.Tuple[str],
+                 only_user: typing.Tuple[str],
+                 start_time: datetime,
+                 duration,
+                 cache_dir,
+                 log_level,
+                 sample_rate_s,
+                 **kwargs):
+
+    da = DeriveArgs(skip_user,
+                    only_user,
+                    start_time,
+                    duration,
+                    log_level,
+                    cache_dir,
+                    tracking_uri=kwargs["tracking_uri"],
+                    source="azure")
+
+    da.init()
+
+    config: Config = get_ae_config(labels_file="data/columns_ae_azure.txt",
+                                   userid_column_name="username",
+                                   timestamp_column_name="timestamp")
+
+    # Specify the column names to ensure all data is uniform
```

Comment:
Can you pull source_column_info, preprocess_column_info, and the module config in another file, just so its easier to digest the workflow quickly, and then if someone wants/needs to they can dig into the config.

Reply:
Done.